cpp/src/lists/contains.cu

```diff
@@ -282,8 +432,13 @@ std::unique_ptr<column> contains(lists_column_view const& lists,
                                  rmm::cuda_stream_view stream,
                                  rmm::mr::device_memory_resource* mr)
 {
-  return to_contains(
-    index_of(lists, search_key, duplicate_find_option::FIND_FIRST, stream), stream, mr);
+  return to_contains(index_of(lists,
```

Comment:
I'd like to see this split into two function calls on separate lines. It makes it less obvious how memory resources are used for intermediates when the functions are combined. Same below.